bakerydemo/recipes/models.py

```diff
@@ -0,0 +1,152 @@
+from django.db import models
+from modelcluster.fields import ParentalKey
+from wagtail.admin.panels import FieldPanel, HelpPanel, InlinePanel, MultiFieldPanel
+from wagtail.fields import RichTextField, StreamField
+from wagtail.models import Orderable, Page
+from wagtail.search import index
+
+from bakerydemo.base.blocks import BaseStreamBlock
+from bakerydemo.base.models import Person
+
+from .blocks import RecipeStreamBlock
+
+
+class RecipePersonRelationship(Orderable, models.Model):
+    """
+    This defines the relationship between the `Person` within the `base`
+    app and the RecipePage below. This allows people to be added to a RecipePage.
+
+    We have created a two way relationship between RecipePage and Person using
+    the ParentalKey and ForeignKey
+    """
+
+    page = ParentalKey(
+        "RecipePage",
+        related_name="recipe_person_relationship",
+        on_delete=models.CASCADE,
+    )
+    person = models.ForeignKey(
+        "base.Person",
+        related_name="person_recipe_relationship",
+        on_delete=models.CASCADE,
+    )
+    panels = [FieldPanel("person")]
+
+
+class RecipePage(Page):
```

Comment:
Additions compared to BlogPage:

- `block_counts` validation for specific block types
- `help_text` for StreamField
- Rich text field with limited set of formatting options.
- HelpPanel
- StreamField inside a MultiFieldPanel
- `title` FieldPanel outside StreamField